curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-dom-manipulation-by-building-a-role-playing-game/62a23c1d505bfa13747c8a9b.md

```diff
@@ -1,8 +1,8 @@
 ---
 id: 62a23c1d505bfa13747c8a9b
-title: Step 17
+title: Step 11
 challengeType: 0
-dashedName: step-17
+dashedName: step-11
```

Comment:
I think it would be better to open up a separate issue for this and have a discussion with the JS team on possibly making changes.  Then if the consensus is to make changes it can be handled in a separate PR. 

Even though this is a large PR with a lot of changes, I do want to minimize adding even more changes like updating portions of the seed code if we can help it. It would be better to have a separate PR for those changes so it is cleaner for reviews 👍